### DISCUSSION TITLE
Parameters passed to the wizard is not stored in the state

### DISCUSSION THREAD

**Discussion by rabol** [spatie/laravel-livewire-wizard#75] [Q&A] [answered]
Hi

I have created a wizard for user account creation, and I pass in a 'type' in my blade file like this:

```
<livewire:registration-wizard :type="$type" />
```

in the 'main' wizard class, the public property is set, but it is not stored in the state

is that a bug or as expected ?

**Comment by ju5t** [comment, marked as answer]
That is as expected, as it's now how you set the initial state. See https://spatie.be/docs/laravel-livewire-wizard/v1/usage/setting-initial-state.
